README.rst

```diff
@@ -41,16 +41,22 @@ Blockstore was originally developed by Harvard's  `LabXchange <https://www.labxc
 Current Status
 --------------
 
-Blockstore is currently implemented as an independently deployed application (IDA), and is used to power `Content Libraries v2 <https://github.com/openedx/frontend-app-library-authoring#readme>`_ as well as `LabXchange <https://www.labxchange.org/>`_.
+As of **April 2022** ("master" version of Open edX) or the **Nutmeg** named release of Open edX, Blockstore is included as a django app within Open edX platform. See `moving blockstore into edx-platform <decisions/0002-app-not-service.rst>`_ for an explanation of this. Prior to those versions, Blockstore was only available as an independent application, and was not included in any Open edX installations by default.
 
-Blockstore is **not** included by default in a standard installation of Open edX or Open edX devstack. However, we are currently (April 2022) `moving blockstore into edx-platform <decisions/0002-app-not-service.rst>`_ - see https://github.com/openedx/edx-platform/pull/29779 for the current status of that work.
+Blockstore is used to power the **Content Libraries v2** feature of Open edX. It is *not* directly used for course functionality, nor for the "v1" Content Libraries feature.
 
 --------------
 Design Details
 --------------
 
 See `DESIGN <DESIGN.rst>`_ for an overview of Blockstore's design as it exists today. See `"Blockstore Design" <https://openedx.atlassian.net/wiki/spaces/AC/pages/737149430/Blockstore+Design>`_ on the wiki for historical context.
 
+------------------------------------------------
+Using with Content Libraries on a Tutor Devstack
+------------------------------------------------
+
+The easiest way to try out the "Content Libraries v2" feature along with Blockstore is to use the Tutor devstack and
+`this library-authoring MFE Tutor plugin <https://github.com/openedx/frontend-app-library-authoring/pull/50>`_. See that plugin's README for details.
 
 --------------------------
 Using with Docker Devstack
```

Comment:
These instructions are for running blockstore as a service, incidentally alongside the docker devstack. Since we can now run blockstore as an app, and the docker devstack is deprecated, do we even need these instructions anymore?

But if we're keeping them, they should be updated. E.g. just below here, it says:

> Prerequisite: Have your Open edX `Devstack <https://github.com/openedx/devstack>`_ properly installed and working. Your devstack must be tracking the ``master`` branch of ``edx-platform``; using Blockstore on an older devstack release is not supported.

But above, we say that it's supported from nutmeg onwards.

The notes about running integration tests are outdated as well, and are ok to remove.

Reply:
The preface for "Using in Production" can be changed to:

> By default, blockstore is run as an app inside of Open edX. Enable it using the waffle switch [`blockstore.use_blockstore_app_api`](https://github.com/openedx/edx-platform/blob/master/openedx/core/lib/blockstore_api/config/waffle.py).
> 
> If you need to run blockstore as a separate service (e.g. for scalability or performance reasons), you can deploy blockstore in production using the `blockstore ansible role <https://github.com/openedx/configuration/tree/master/playbooks/roles/blockstore>`_.
> ...

Reply:
Thanks @pomegranited ! I pushed 575c37effe9a386612e0bef06c9ed8990a2f60fa with those changes. Let me know your thoughts please :)